src/config.ts

```diff
@@ -139,6 +139,12 @@ export const AUTH0_CLIENT_ID =
   // dev app client id as fallback for e2e tests
   'YgsHPq93Egfap5Wc4iEQlGyQMqjLeBf2'
 
+export const HASHED_KEYSHARE_PEPPER = keyOrUndefined(
```

Comment:
Spoke with Satish a bit about this (using what is essentially public information as a pepper)... I'm actually not sure this technically counts as _either_ a pepper _or_ a salt in this case, since the only criterion for a pepper is:
* they should be a secret
while the only criterion for a salt is:
* they should be unique among all salts used in the system

This doesn't meet either of those (if we consider it public info), since its lack of secrecy disqualifies it from being a pepper, and its lack of uniqueness disqualifies it from being a salt. I'm not really sure if this matters, but I'm also not enough of an expert in security to know whether or not prefixing/suffixing all secrets with a fixed, public string is in anyway a detriment to the security of the system.

It's prolly fine, but I think I'd err on the side of caution and just drop the idea of using a suffix/prefix at all - it keeps the code simpler, anyhow. May want to check with @nategraf on the security implications of this.

Reply:
that makes sense, I think I'm a bit confused about how the encrypted secrets.json file works in the wallet then, are those secrets public information if you don't have google access? If so, why are they stored in the secrets file?

Reply:
My understanding is that the encryption process is a sort of best-effort at obfuscating the secrets (i.e., not _literally_ checking the secrets in plaintext into version control) but typically I consider anything that's _used_ by a front-end, even if only in-memory (and not checked into/hardcoded into the source), to be public information.